package.json

```
@@ -60,6 +60,14 @@
           "default": false,
           "hint": "Secret to append to preview link query parameter, leave blank if none is needed"
         },
+        {
+          "id": "disablePreviewPath",
+          "label": "(Optional) Disable preview API path",
+          "type": "string",
+          "required": false,
+          "default": "/api/disable-preview",
```

I haven't actually tested this, but will this make `disablePreviewPath` be `"/api/disable-preview"` even if you don't specify it? Or does this just provide a placeholder? If the former, this should probably be
```suggestion
          "default": "",
```